dataactcore/models/jobTrackerInterface.py

```diff
@@ -382,3 +384,24 @@ def checkJobType(self, jobId):
             # Wrong type
             raise ResponseException("Wrong type of job for this service", StatusCode.CLIENT_ERROR, None,
                                     ValidationError.jobError)
+
+    def getTotalObligations(self,submissionId):
+        """ Return sum of all obligations incurred """
+        query = self.session.query(func.sum(AwardFinancial.transaction_obligated_amou)).filter(AwardFinancial.submission_id == submissionId).scalar()
+        if query == None:
```

Comment:
Small thing to point out here and in the other new methods...the preferred way to check for `None` in Python is `if query is None` 

(reference: https://www.python.org/dev/peps/pep-0008/#programming-recommendations...2nd bullet re: comparison to singletons)